src/DurableTask.Netherite/StorageLayer/Faster/StoreWorker.cs

```diff
@@ -320,6 +320,16 @@ bool CheckpointDue(out CheckpointTrigger trigger, out long? compactUntil)
                     trigger = CheckpointTrigger.Idle;
                 }
             }
+
+            var checkpointDueLog = $"CheckpointDue Debug: trigger={trigger}, " +
+                $"lastCheckpointedCommitLogPosition={this.lastCheckpointedCommitLogPosition}, " +
+                $"MaxNumberBytesBetweenCheckpoints={this.partition.Settings.MaxNumberBytesBetweenCheckpoints}," +
+                $"CommitLogPosition={this.CommitLogPosition}, " +
+                $"numberEventsSinceLastCheckpoint={this.numberEventsSinceLastCheckpoint}," +
+                $"MaxNumberEventsBetweenCheckpoints={this.partition.Settings.MaxNumberEventsBetweenCheckpoints}," +
+                $"inputQueuePositionLag={inputQueuePositionLag}," +
+                $"timeOfNextIdleCheckpoint={this.timeOfNextIdleCheckpoint}";
+            this.traceHelper.FasterProgress(checkpointDueLog);
```

Comment:
This will generate A LOT of log messages so we won't want to merge this into the release but it is probably fine for a troubleshooting exercise.

Reply:
When you say "this" you mean this particular line, or the logs across this PR in general?

Reply:
just this line. The others fire only as the checkpoint state machine progresses, which is not that frequent. But this line fires every time the storeworker goes through its loop body.

Reply:
I opted to remove this. This generates ~5k logs per second per worker, which could cause ingestion problems or even overload the VM. I'm gonna look to make this log more periodic in a future iteration